pages/entry.jsx

```diff
@@ -66,22 +66,28 @@ export default React.createClass({
   },
   render() {
     let docTitle;
+    let description;
     let justPostedByUserMessage = null;
     let content;
 
     if (this.state.dataLoaded) {
       docTitle = `${this.state.entry.title}`;
       justPostedByUserMessage = this.state.justPostedByUser ? (<h5 className="col-12 text-center">Thanks for submitting!</h5>) : null;
       content = <ProjectCard {...Utility.processEntryData(this.state.entry)} onDetailView={true} />;
+      description = `${this.state.entry.description}`;
```

Comment:
There is no reason for template string syntax here: we're not templating anything into another string. Just use `description = this.state.entry.description;`